Mu2eG4/geom/protonAbsorber_cylindrical_v04.txt

```diff
@@ -0,0 +1,128 @@
+//
```

Comment:
What is the difference between this and v03?  is it just the overlap avoidance?

Reply:
Yes, I just changed the inner radius of two OPA supports (OPASupport_4 and _5).
I didn't do the including of the previous version + changed parameter style, as it was decided this could be
confusing/messy. I instead copied all the previous configuration, and then changed just this parameter.

Reply:
I looked at the diff more carefully - there are a lot of other lines changed, not just the two radius changes.  

Is that what you expected me to see?

Reply:
OK.  I think I understand now.  You made changes to this file, independent of the radius change.  Then you made the radius change.  Is that right?

Reply:
The support slat lines are from protonAbsorber_cylindrical_v02, where I've input the text that was included
into v03 by the #include at the top. I just tested this by putting the text of v02 and then v03 into a new file, removing all
entries from v02 replaced by v03, and then comparing the result to v04. The only differences I got were the OPA
support radii and the overall and outer proton absorber version, which I incremented to match the file version number.

Let me know if you find that I've missed something, and the parameters have changed!